src/main/java/it/gov/pagopa/apiconfig/core/controller/IbanController.java

```diff
@@ -494,4 +495,63 @@ public ResponseEntity<Void> massiveCreateIbans(
 	    ibansService.createMassiveIbans(file);
 	    return ResponseEntity.status(HttpStatus.CREATED).build();
 	  }
+
+
+    @Operation(
+            summary = "Create or update a label to be associated to IBANs",
+            security = {
+                    @SecurityRequirement(name = "ApiKey"),
+                    @SecurityRequirement(name = "Authorization")
+            },
+            tags = {
+                    "Iban",
+            })
+    @ApiResponses(
+            value = {
+                    @ApiResponse(
+                            responseCode = "200",
+                            description = "OK",
+                            content = @Content(
+                                    mediaType = MediaType.APPLICATION_JSON_VALUE,
+                                    schema = @Schema(implementation = IbanLabel.class))),
+                    @ApiResponse(
+                            responseCode = "400",
+                            description = "Bad Request",
+                            content =
+                            @Content(
+                                    mediaType = MediaType.APPLICATION_JSON_VALUE,
+                                    schema = @Schema(implementation = ProblemJson.class))),
+                    @ApiResponse(
+                            responseCode = "401",
+                            description = "Unauthorized",
+                            content = @Content(schema = @Schema())),
+                    @ApiResponse(
+                            responseCode = "403",
+                            description = "Forbidden",
+                            content = @Content(schema = @Schema())),
+                    @ApiResponse(
+                            responseCode = "404",
+                            description = "Not Found",
+                            content =
+                            @Content(
+                                    mediaType = MediaType.APPLICATION_JSON_VALUE,
+                                    schema = @Schema(implementation = ProblemJson.class))),
+                    @ApiResponse(
+                            responseCode = "429",
+                            description = "Too many requests",
+                            content = @Content(schema = @Schema())),
+                    @ApiResponse(
+                            responseCode = "500",
+                            description = "Service unavailable",
+                            content =
+                            @Content(
+                                    mediaType = MediaType.APPLICATION_JSON_VALUE,
+                                    schema = @Schema(implementation = ProblemJson.class)))
+            })
+    @PostMapping(
+            value = "/ibans/labels",
+            produces = {MediaType.APPLICATION_JSON_VALUE})
+    public ResponseEntity<IbanLabel> upsertIbanLabel(@RequestBody @Valid @NotNull IbanLabel ibanLabel) {
```

Comment:
`upsert` logic is not standard for `apiconfig`. do you think it is a food choice?

Reply:
Yes, I know it is not a standard API in `apiconfig`. But, for simplicity, I decided to use a single API that can:
- create a new label, if one with the same name doesn't exists yet
- overwrite an existing one, if one already exists with the same name
Since this is a configuration API, used only when a new IBAN label is needed, this can help us to avoid using two different APIs if a label description is incorrect.